Resources/views/Collector/db.html.twig

```diff
@@ -409,5 +410,81 @@
+                    <img alt="+" src="data:image/gif;base64,R0lGODlhEgASAMQTANft99/v+Ga44bHb8ITG52S44dXs9+z1+uPx+YvK6WC24G+944/M6W28443L6dnu+Ge54v/+/l614P///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAABMALAAAAAASABIAQAVS4DQBTiOd6LkwgJgeUSzHSDoNaZ4PU6FLgYBA5/vFID/DbylRGiNIZu74I0h1hNsVxbNuUV4d9SsZM2EzWe1qThVzwWFOAFCQFa1RQq6DJB4iIQA7" style="display: inline; width: 12px; height: 12px;" />
+                    <img alt="-" src="data:image/gif;base64,R0lGODlhEgASAMQSANft94TG57Hb8GS44ez1+mC24IvK6ePx+Wa44dXs92+942e54o3L6W2844/M6dnu+P/+/l614P///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAABIALAAAAAASABIAQAVCoCQBTBOd6Kk4gJhGBCTPxysJb44K0qD/ER/wlxjmisZkMqBEBW5NHrMZmVKvv9hMVsO+hE0EoNAstEYGxG9heIhCADs=" style="display: none; width: 12px; height: 12px;" />
+                    <span style="vertical-align:top">Explain query</span>
+                </a>]
+                {% else %}
+                    This query cannot be explained
+                {% endif %}
+
+                {% if query.explainable %}
+                    <div id="explain-{{ i }}-{{ loop.parent.loop.index }}" class="loading"></div>
+                {% endif %}
+            </td>
+        </tr>
+    {% endfor %}
+{% endblock %}
+
+{% block query_table_v2 %}
+    {% for i, query in queries %}
+        <tr id="queryNo-{{ i }}-{{ loop.parent.loop.index }}">
+            <td>{{ loop.index }}</td>
+            <td class="nowrap">{{ '%0.2f'|format(query.executionMS * 1000) }} ms</td>
+            <td>
+                {{ query.sql|doctrine_highlight_sql }}
+
+                <div>
+                    <strong class="font-normal text-small">Parameters</strong>: {{ query.params|yaml_encode }}
+                </div>
+
+                <div class="text-small font-normal">
+                    <a href="#" id="expandParams-{{ i }}-{{ loop.parent.loop.index }}" class="sf-toggle link-inverse" data-toggle-selector="#original-query-{{ i }}-{{ loop.parent.loop.index }}" data-toggle-alt-content="Hide runnable query">View runnable query</a>
+
+                    {% if query.explainable %}
+                        &nbsp;&nbsp;
+                        <a class="link-inverse" href="{{ path('_profiler', { panel: 'db', token: token, page: 'explain', connection: connection, query: i }) }}" onclick="return explain(this);" data-target-id="explain-{{ i }}-{{ loop.parent.loop.index }}">Explain query</a>
+                    {% endif %}
+                </div>
+
+                <div id="original-query-{{ i }}-{{ loop.parent.loop.index }}" class="sql-runnable hidden">
+                    {{ (query.sql ~ ';')|doctrine_replace_query_parameters(query.params, highlight = false)|doctrine_highlight_sql }}
```

Comment:
In some apps, when you don't add the `;` you cannot run the query. In contrast, adding it doesn't prevent any app to run the query (as far as I know).